### PR TITLE
add android auto carinfo documentation

### PR DESCRIPTION
**Pull request by drosoCode**
Add Android Auto CarInfo sensors documentation

Android PR: https://github.com/home-assistant/android/pull/3399

### CODE REVIEW

**Comment by dshokouhi** [comment]
Please also add the beta label to the new sensors so users can distinguish between what is in production and the beta version.

File: docs/core/sensors.md [hunk 1]
```diff
@@ -177,9 +177,23 @@ The attribute for the state will reflect the `confidence` rating from the [Activ
...
-## Android Auto
+## Android Auto Sensors
 ![Android](/assets/android.svg)
-This sensor is used to determine if the device is connected to Android Auto.  The attributes will return the specific type of connection.
+The connection sensor is used to determine if the device is connected to Android Auto. The attributes will return the specific type of connection.
+
+The CarInfo sensors listed below describe the state of the car for a few different data points. These sensors may not be available depending on your Android Auto hardware/software and on your phone software.
+
+:::caution
+ Note that you need to start the Home-Assistant app on your Android Auto screen each time that you connect the phone to the car to allow these sensors to work (once started, you can dismiss the app). By default, if you have one of these sensors enabled, a notification will be shown when you connect you phone to start the app. You can disable this by muting the "Open Android Auto App" notification channel.
```
Comment (dshokouhi) [inline docs/core/sensors.md:187]: May want to fix this paragraph to mention the new `car_ui` parameter :)

File: docs/core/sensors.md [hunk 2]
```diff
@@ -177,9 +177,23 @@ The attribute for the state will reflect the `confidence` rating from the [Activ
...
+:::
+
+|| Sensor | Description |
+| --------- | --------- |
+| `android_auto_battery_level` | <span class='beta'>BETA</span> The percentage of battery remaining |
+| `android_auto_car_name` | <span class='beta'>BETA</span> The name of the car. The manufacturer name and manufactured year are in the attributes |
+| `android_auto_car_status` | <span class='beta'>BETA</span> The charging status of the car (only for EVs). The state of the charging port is in the attributes  |
+| `android_auto_fuel_level` | <span class='beta'>BETA</span> The percentage of fuel remaining |
+| `android_auto_odometer` | <span class='beta'>BETA</span> The the value of the car odometer in meters |
```
Comment (dshokouhi) [inline docs/core/sensors.md:196]: We should update the sensor names to match the latest PR changes

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Thanks for adding the beta label. Just a couple comments to bring the docs up to line with current features and latest PR changes. Also need to resolve the conflict in the sensors file.

**Review by dshokouhi — APPROVED** [review]
Looks good to me, can be merged when parent is merged

File: docs/core/sensors.md [hunk 3]
```diff
@@ -306,6 +307,24 @@ A Monitor setting toggle will start or stop the scans - this setting can also be
...
 When the app is actively scanning for beacons a notification will be shown to make background scanning more reliable. If you are on Android 8.0+ you are free to minimize and/or turn off the notification channel for the `Beacon Monitor Scanning`.
 
+## Car Sensors
+![Android](/assets/android.svg)
+
+The sensors listed below describe the state of the car for a few different data points. Currently this is only available for Android Auto (but these sensors may not be available depending on your phone software).
+
+:::caution
+ Note that you need to start the Home-Assistant app on your Android Auto screen each time that you connect the phone to the car to allow these sensors to work (once started, you can dismiss the app).
+ To make things easier, you may want to use the [`car_ui` parameter](../android-auto/android-auto.md#notifications) to show a notification on your car when you [connect your phone](./sensors.md#android-auto).
```
Comment (jpelgrom) [inline docs/core/sensors.md:317]: Did you want a visible line break/two paragraphs here? If so you need to keep one line empty. Currently it is one paragraph.

File: docs/core/sensors.md [hunk 4]
```diff
@@ -306,6 +307,24 @@ A Monitor setting toggle will start or stop the scans - this setting can also be
...
+:::
+
+|| Sensor | Description |
```
Comment (jpelgrom) [inline docs/core/sensors.md:320]: Typo results in the first column having no title, second column titled Sensor, and Description being dropped from the table

```suggestion
| Sensor | Description |
```

**Review by jpelgrom — COMMENTED** [review]
Should this be referred to on the Android Auto page as well?

**Pull request merged**